semantics/language/common/error.k

```diff
@@ -7,7 +7,7 @@ module C-ERROR-SYNTAX
      syntax KItem ::= UNDEF(String, String, String)
      syntax KItem ::= UNSPEC(String, String, String)
      syntax KItem ::= CV(String, String, String)
-
+     syntax KItem ::= IMPL(String, String, String)
```

Comment:
these need to be modified to add the prefixes to the error messages, and then the output files updated.